java/src/server/plugin/PluginLoader.java

```diff
@@ -1,23 +1,30 @@
 package server.plugin;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class PluginLoader implements IPluginLoader {
 
-
+    /**
+     * @inheritDoc
+     */
     @Override
     public List<IPlugin> loadConfig(List<PluginConfig> configs) {
         return null;
     }
 
     @Override
     public List<IPlugin> startPlugins(List<IPlugin> plugins) {
-        return null;
+        return plugins.stream()
+                .map(IPlugin::start)
+                .collect(Collectors.toList());
```

Comment:
If `IPlugin.start` (and `stop`) return `null` on error, this will return a list of plugins where some may be null. What's the return value of `startPlugins` supposed to represent? The plugins that were started? If that's the case, you should probably add a `.filter(Objects::nonNull)`.